### PR TITLE
[FIX] update plot_surface_image_and_maskers.py

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none

Changes proposed in this pull request:

- remove duplicate code
- add more explanations
- use more consistent code in the example

### CODE REVIEW

File: examples/07_advanced/plot_surface_image_and_maskers.py [hunk 1]
```diff
@@ -81,17 +113,9 @@
...
 fsaverage = load_fsaverage("fsaverage5")
 destrieux = fetch_atlas_surf_destrieux()
-labels_img = SurfaceImage(
-    mesh=fsaverage["pial"],
-    data={
-        "left": destrieux["map_left"],
-        "right": destrieux["map_right"],
-    },
-)
-label_names = [x.decode("utf-8") for x in destrieux.labels]
```
Comment (Remi-Gau) [inline examples/07_advanced/plot_surface_image_and_maskers.py:91]: code duplication

File: examples/07_advanced/plot_surface_image_and_maskers.py [hunk 2]
```diff
@@ -104,18 +128,10 @@
...
-print(f"Destrieux image: {labels_img}")
-plot_surf_roi(
-    roi_map=labels_img,
-    avg_method="median",
-    view="lateral",
-    bg_on_data=True,
-    bg_map=fsaverage_sulcal,
-    darkness=0.5,
-    title="Destrieux atlas",
-)
```
Comment (Remi-Gau) [inline examples/07_advanced/plot_surface_image_and_maskers.py:116]: this already exist in another example

**Comment by Remi-Gau** [comment]
[HTML version of example](https://output.circle-artifacts.com/output/job/5f970dde-2928-423c-8c4d-a46f057d8ac5/artifacts/0/dev/auto_examples/07_advanced/plot_surface_image_and_maskers.html#sphx-glr-auto-examples-07-advanced-plot-surface-image-and-maskers-py)

**Review by bthirion — APPROVED** [review]
The PR LGTM, but I think that in the long term we should have a decoding example that makes sense, rather than random labels on random data.

**Comment by emdupre** [comment]
This LGTM ! Only one small request, if you have a cycle : the labels on this connective plot are hard to read (at least in mobile ?) :

image

Any possibility of plotting every other, or every e.g. fifth to improve legibility ?

**Comment by Remi-Gau** [comment]
> the labels on this connective plot are hard to read

will keep this for 0.11.1

**Pull request merged**